backend/src/services/user-service.ts

```
@@ -0,0 +1,79 @@
+    const [results]: [RowDataPacket[], any] = await db.query(query, sessionId);
+    return results[0].count > 0;
+  }
+
+  static async login(userId: number, password: string) {
+    const query = "SELECT Password FROM Customers WHERE user_id = ?";
+    const [results]: [RowDataPacket[], any] = await db.execute(query, userId);
+    const storedHash = results[0].Password; // Notice capital P if it's case-sensitive
+
+    const isMatch = await bcrypt.compare(password, storedHash);
+    if (isMatch) {
+      const sessionid = await this.createOrUpdateSession(userId);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)